tests/post-command.bats

```diff
@@ -5,17 +5,36 @@ load "$BATS_PLUGIN_PATH/load.bash"
 # Uncomment the following line to debug stub failures
 # export BUILDKITE_AGENT_STUB_DEBUG=/dev/tty
 
-setup() {
+
+setup () {
   export BUILDKITE_PLUGIN_WIZ_SCAN_TYPE="docker"
   export BUILDKITE_PLUGIN_WIZ_IMAGE_ADDRESS="ubuntu:22.04"
   export WIZ_DIR="$HOME/.wiz"
 }
 
+@test "Captures docker exit code and exits plugin when non-0 status" {
+
+  export WIZ_API_ID="test"
+  export WIZ_API_SECRET="secret"
+
+  stub docker : 'exit 1'
+  mkdir -p "$WIZ_DIR"
+  touch "$WIZ_DIR/key"
+
+  run "$PWD/hooks/post-command"
+  #todo test docker scan
+  assert_failure
+  #cleanup
+  rm "$WIZ_DIR/key"
+}
+
 @test "Authenticates to wiz using \$WIZ_API_SECRET" {
   export WIZ_API_ID="test"
   export WIZ_API_SECRET="secret"
 
-  stub docker 'echo TODO'
+  stub docker : 'exit 0'
+  stub docker : 'exit 0'
+  stub docker : 'exit 0'
```

Comment:
As a stub I don't believe this needs to be defined multiple times

Reply:
The tests failed if I didn't stub multiple times for that particular test, and it's really not clear why :(